src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const CopyState = { ...state };
```

Comment:
do not use capital letters as the first letter in variable names
```suggestion
  const сopyState = { ...state };
```